### PR TITLE
Allow alternate tau dir for tests

### PR DESCRIPTION
**Pull request by wspear**

### CODE REVIEW

File: README.md
Comment (zbeekman) [inline README.md:54]: @wspear what was the motivation for changing this?

File: README.md
Comment (zbeekman) [inline README.md:115]: So the tests pass when setting -DTAU_ROOT? Perhaps we should update the paragraph above too. Right not is reads like:

>This doesn't work. Ok, or you can try this...

I.e. a bit self contradictory. I can help with wording.

**Review by zbeekman — COMMENTED** [review]
@wspear this generally looks good. After responding to my quick questions I'll approve and merge or we can make some small tweaks if needed.

File: README.md
Comment (wspear) [reply README.md:54]: At least as of docker 25.0.3 --pull requires an argument or else you get: `invalid pull option: '--tmpfs=/dev/shm:rw,nosuid,nodev,exec': must be one of "always", "missing" or "never".`

**Review by zbeekman — APPROVED** [review]
LGTM assuming tests all pass

**Pull request merged**
